### ISSUE TITLE
vulnerability in base Alpine OS

### ISSUE THREAD

**Issue by fidesmo-mrc** [anapsix/docker-alpine-java#73]
there are several RCE bugs in apk-tools: https://justi.cz/security/2018/09/13/alpine-apk-rce.html

the bugs are fixed and the Alpine OS images updated,

Would it be possible to upgrade alpine to the latest 3.7 version?

thanks!

**Comment by anapsix** [comment]
thanks for bringing this up, I've triggered a rebuild of latest and `8u181b13_*` tags

**Comment by doublefx** [comment]
Hi, I was about to create an issue but seen this one, it seems to me the fix is in the 3.8.1, not in 3.7, see: 

- https://alpinelinux.org/posts/Alpine-3.8.1-released.html
- https://git.alpinelinux.org/cgit/apk-tools/commit/?id=6484ed9849f03971eb48ee1fdc21a2f128247eb1

**Comment by anapsix** [comment]
Just for kicks, I've added a Java8 server-jre Dockerfile to Quay, here are [vulnerability scan results](https://quay.io/repository/anapsix/alpine-java/manifest/sha256:d8684d2a9b555434616ba1fe2a93fa546453692295976c188778159c790bb0df?tab=packages)

Do you think we are good?
```
$ docker images alpine:3.7
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
alpine              3.7                 34ea7509dcad        6 days ago          4.21MB

$ docker run -it --rm alpine:3.7 sh -c 'apk list apk-tools'
WARNING: Ignoring APKINDEX.70c88391.tar.gz: No such file or directory
WARNING: Ignoring APKINDEX.5022a8a2.tar.gz: No such file or directory
apk-tools-2.10.1-r0 x86_64 {apk-tools} (GPL2) [installed]
```
